util/query-helper.js

```diff
@@ -202,17 +202,33 @@ async function getAgendaInformation(latestAgendaURI) {
     PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
     PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
     PREFIX dct: <http://purl.org/dc/terms/>
+    PREFIX prov: <http://www.w3.org/ns/prov#>
+    PREFIX generiek:  <https://data.vlaanderen.be/ns/generiek#>
 
     SELECT DISTINCT ?planned_start ?data_docs ?publication_date ?kind WHERE {
       GRAPH <${targetGraph}> {
         ${sparqlEscapeUri(latestAgendaURI)} a besluitvorming:Agenda ;
           besluitvorming:isAgendaVoor ?meeting .
         ?meeting besluit:geplandeStart ?planned_start .
-        OPTIONAL { ?meeting ext:algemeneNieuwsbrief ?newsletter . }
         OPTIONAL { ?meeting dct:type ?kind }
-        OPTIONAL { ?newsletter ext:issuedDocDate ?data_docs . }
-        OPTIONAL { ?newsletter dct:issued ?publication_date . }
+        OPTIONAL {
+          SELECT ?meeting MIN(?publicationTime) AS ?publication_date WHERE {
+            ?meeting a besluit:Vergaderactiviteit .
+            ?meeting ^prov:used ?themisPublicationActivity .
+            ?themisPublicationActivity a ext:ThemisPublicationActivity .
+            OPTIONAL { ?themisPublicationActivity generiek:geplandeStart ?plannedPublicationTime . }
+            ?themisPublicationActivity ext:onbevestigdePublicatietijd ?unconfirmedPublicationTime .
+            BIND (COALESCE(?plannedPublicationTime, ?unconfirmedPublicationTime) AS ?publicationTime)
```

Comment:
Te bespreken met Tom, Ik heb momenteel nog niet het nut gezien van de "unconfirmed" datum. Dit maakt alles wat lastiger omdat er 2 datums zijn dat je mogelijk moet vergelijken.

Reply:
Ook zou "unconfirmed" betekenen dat de gebruikers niet bevestigd hebben wanneer de documenten moeten worden vrijgegeven. Dan lijkt het me best dat we ook geen mogelijk foutieve datum gaan de wereld insturen.
Dan lijkt het me eerder logisch dat we in de frontend afdwingen dat bij uitsturen van de nieuwsbrief we eerst de gebruiker gaan prompten om de datum vrijgave documenten te bevestigen.

Mijn huidig voorstel is om die "unconfirmed" er even uit te halen. Toch alvast uit de newsletter-service.